tests/js-units/data/display-in-legend-capabilities.json

```diff
@@ -357,6 +357,70 @@
                     "opaque": false,
                     "noSubsets": false
                 },
+                {
+                    "Name": "group-without-children",
+                    "Title": "group-without-children",
+                    "CRS": [
+                        "CRS:84",
```

Comment:
Why repeat?

Reply:
I don't know it is generated by OpenLayers from the WMS GetCapabilities